packages/openapi/src/openapi-request-builder.spec.ts

```diff
@@ -76,7 +76,28 @@ describe('openapi-request-builder', () => {
     expect(response.data).toBe(dummyResponse);
   });
 
-  it('executeRaw executes a request with header parameters', async () => {
+  it('executes a request without parameters and basePath explicitly set using executeRaw', async () => {
+    const requestBuilder = new OpenApiRequestBuilder(
+      'get',
+      '/test'
+    ).setBasePath('/base/path/to/service');
+    const response = await requestBuilder.executeRaw(destination);
+    expect(httpClient.executeHttpRequest).toHaveBeenCalledWith(
+      sanitizeDestination(destination),
+      {
+        method: 'get',
+        middleware: [],
+        url: 'base/path/to/service/test',
```

Comment:
@tomfrenken Fyi since it relates to some of your code changes like two years ago. Here without a leading slash is the same behaviour as in OData, where remove (leading/trailing) slashes functions were used at almost each and every step to ensure no slashes at all. But it feels kind inconsistent to me because without setting the base path, we use and probably also recommend adding a leading slash at the path.